lib/GeoExt/data/MapFishPrintv3Provider.js

```diff
@@ -321,6 +321,9 @@ GeoExt.data.MapFishPrintv3Provider = Ext.extend(GeoExt.data.PrintProviderBase, {
                     matrices: []
                 });
                 if (layer.requestEncoding === "REST") {
+                    enc.baseURL = enc.baseURL
+                        .replace(/%7B/g, '{')
+                        .replace(/%7D/g, '}');
```

Comment:
I look a it around and finally my question is why do we have a `getAbsoluteUrl` call in the HTTPRequest encoder:
https://github.com/geoext/geoext/blob/master/lib/GeoExt/data/PrintProviderBase.js#L736
He come from here: https://github.com/geoext/geoext/commit/d20af6333a8900dfb95bb9885e57328fd839ff6c
I understand why we have it for Image layer, for the legend, in the style, but not on the HTTPRequest Layer Is there a reason for it, can we remove it ? @bartvde, @ahocevar do you have an opinion ?